packages/core/adder/options.ts

```diff
@@ -113,7 +113,7 @@ export function prepareAndParseCliOptions<Args extends OptionDefinition>(adderDe
     }
 
     if (multipleAdders) {
-        program.option("--adder <string...>", "List of adders to install");
+        program.argument("<adders...>", "List of adders to install");
```

Comment:
this will probably fix the issue @manuel3108 ran into
```suggestion
        program.argument("[adders...]", "List of adders to install");
```

Reply:
Let''s give it a try!